speculative_decoding/engine_building/build_mistral.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+
+from huggingface_hub import snapshot_download
+
+# MODEL_NAME = "mistral-7B"
+# MODEL_HF_ID = "mistralai/Mistral-7B-v0.1"
+# Mixtral gets killed...
+MODEL_NAME = "mixtral-7B"
+MODEL_HF_ID = "mistralai/Mixtral-8x7B-v0.1"
```

Comment:
I think this should be https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2. @aspctu could you confirm?